catatonit.c

```diff
@@ -361,6 +439,8 @@ int main(int argc, char **argv)
 		bail("self-check that pid1 (%d) was spawned failed: %m", pid1);
 	debug("pid1 (%d) spawned: %s", pid1, argv[0]);
 
+	close_fds_ge_than(3, sfd);
```

Comment:
We obviously shouldn't bail here, but some kind of debug output if closing failed wouldn't hurt.

Reply:
there is already a warn for each `close()` that fails in the fallback code.  Should we add another warn here?

Reply:
If close_range() is supported you wouldn't get any indication the close failed right? It could just be a debug message so you don't get duplicate warnings.